src-v2/NTS.Persistance/Adapters/SetupLapRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using NTS.Domain.Setup.Entities;
+using NTS.Persistence.Setup;
+
+namespace NTS.Persistence.Adapters;
+
+public class SetupLapRepository : SetRepository<Lap, SetupState>
+{
+    public SetupLapRepository(IStore<SetupState> store) : base(store)
```

Comment:
I don't think the `Setup` prefix is necessary here unless we decide to prefix all our repositories. Currently I'm only prefixing duplicate names and there isn't a `Lap` nor `Loop` in Core.